src/main/java/de/retest/recheck/Properties.java

```diff
@@ -49,4 +57,18 @@ public static FileOutputFormat getStateOutputFormat() {
 		}
 		return FileOutputFormat.PLAIN;
 	}
+
+	public static Double getConfiguredDouble( final String property, final Double defaultValue ) {
+		final String value = System.getProperty( property );
+		if ( value == null ) {
+			return defaultValue;
+		}
+		try {
+			return Double.parseDouble( value );
+		} catch ( final NumberFormatException e ) {
+			logger.error( "Exception parsing value {} of property {} to double, using default {} instead.", value,
+					property, defaultValue );
+			return defaultValue;
+		}
```

Comment:
`NumberUtils.toDouble`

Reply:
No, actually we want an error logged, not silently the default value used...